### PR TITLE
AKS: fix the scroll coming when hovering over unicorn in nav-left

### PR DESCRIPTION
**Pull request by yabhis**

### CODE REVIEW

File: _sass/sass/components/nav.sass [hunk 1]
```diff
@@ -73,6 +73,7 @@ a.nav-item:not(.button)
     flex-basis: 0
 
 .nav-left
+  overflow: hidden
```
Comment (kevva) [inline _sass/sass/components/nav.sass]: Should be listed in alphabetical order.

**Comment by sindresorhus** [comment]
@yabhis What browser and OS did you see this in? I haven't seen it before.

Can you add the override to main.scss instead? I don't want to edit these files as they're bundled CSS framework files and should be updateable.

File: _sass/sass/components/nav.sass (hunk 1 above)
Comment (sindresorhus) [reply _sass/sass/components/nav.sass]: @kevva I don't really order alphabetically. When I did CSS a lot before, I used to order layout type props first, and then visual only. I don't see any benefit in doing it alphabetically tbh.

**Comment by kevva** [comment]
@sindresorhus, I see this in Chrome 60.0.3112.113.

**Comment by yabhis** [comment]
@sindresorhus , I see this in Chrome Version 60.0.3112.113 (Official Build) (64-bit)

**Pull request closed without merging**